syncmaster/dto/transfers.py

```diff
@@ -107,6 +107,16 @@ def __post_init__(self):
         self.options.setdefault("if_exists", "replace_overlapping_partitions")
 
 
+@dataclass(kw_only=True)
+class IcebergRESTCatalogS3TransferDTO(DBTransferDTO):
+    type: ClassVar[str] = "iceberg_rest_s3"
+    catalog_name: str
```

Comment:
There this field is filled up? Should user pass it in UI, or its is autogenerated within API/worker?